lib/compilers/glimmer/index.js

```diff
@@ -9,14 +14,31 @@ export default class GlimmerTemplatePrecompiler extends Filter {
     this.extensions = ['hbs'];
     this.targetExtension = 'ts';
     this.rootName = options.rootName;
+    this.glimmerEnv = options.GlimmerENV || {};
+    this.plugins = options.plugins || { ast: [] };
```

Comment:
This is probably ok because there are only AST plugins, but we should probably do the ast defaulting separately in case we add more types in the future.

```js
this.plugins = options.plugins || {};
this.plugins.ast = this.plugins.ast || [];
```

This is done below though, so perhaps all we need to do is ensure `this.plugins` is set to an object?